### PR TITLE
Move to python3 ConfigParser in api.py

### PR DESCRIPTION
**Pull request by jimman2003**
See: https://docs.python.org/3/library/configparser.html

refactor
### Technical

Is it compatable with six's ConfigParser?
### Testing

Try to read a ol.ini
### Stakeholders

@dhruvmanila

### CODE REVIEW

File: openlibrary/api.py [hunk 1]
```diff
@@ -23,9 +23,8 @@
 import web
 import logging
 import requests
-
 import six
-from six.moves.configparser import ConfigParser
+import configparser
```
Comment (cclauss) [inline openlibrary/api.py:27]:
```suggestion
from configparser import ConfigParser
```
and then please revert line 80.

Comment (jimman2003) [reply openlibrary/api.py]: Oh yeah :facepalm:

File: openlibrary/api.py [hunk 2]
```diff
@@ -78,7 +77,7 @@ def autologin(self, section=None):
 
         If environment variable OPENLIBRARY_RCFILE is specified, it'll read that file instead of ~/.olrc.
         """
-        config = ConfigParser()
+        config = configparser.ConfigParser()
```
Comment (cclauss) [inline openlibrary/api.py:80]:
```suggestion
        config = ConfigParser()
```

**Review by cclauss — APPROVED** [review]
LGTM.  Thanks.

**Pull request merged**